articles/migrations/guides/calling-api-with-idtokens.md

```diff
@@ -0,0 +1,254 @@
+
+On the `Legacy (ID Token)` script you can see an implementation of the old approach that gets an ID Token (and then uses it to call the endpoint). On the `Current (Access Token)` script you can see the new approach that gets an Access Token as well.
+
+<div class="code-picker">
+  <div class="languages-bar">
+    <ul>
+      <li class="active"><a href="#with-id-token" data-toggle="tab">Legacy (ID Token)</a></li>
+      <li><a href="#with-access-token" data-toggle="tab">Current (Access Token)</a></li>
+    </ul>
+  </div>
+  <div class="tab-content">
+    <div id="with-id-token" class="tab-pane active">
+      <pre class="text hljs">
+        <code>
+POST https://${account.namespace}/oauth/token
+Content-Type: application/json
+{
+  "grant_type": "password",
+  "username": "USERNAME",
+  "password": "PASSWORD",
+  "scope": "openid",
+  "client_id": "${account.clientId}",
+  "client_secret": "YOUR_CLIENT_SECRET",
+}
+        </code>
+      </pre>
+    </div>
+    <div id="with-access-token" class="tab-pane">
+      <pre class="text hljs">
+        <code>
+POST https://${account.namespace}/oauth/token
+Content-Type: application/json
+{
+  "grant_type": "password",
+  "username": "USERNAME",
+  "password": "PASSWORD",
+  "audience": "https://${account.namespace}/api/v2/",
+  "scope": "openid%20read:current_user",
+  "client_id": "${account.clientId}",
+  "client_secret": "YOUR_CLIENT_SECRET",
+}
+        </code>
+      </pre>
+    </div>
+  </div>
+</div>
+
+The differences that we introduced in order to get an Access Token that can access the Management API are the following:
+- We set the `audience` to `https://${account.namespace}/api/v2/`
+- We asked for the additional scope `read:current_user`
+
+Once you have the Access Token you can use it to call the endpoint. This part remains the same, nothing else changes in the request except for the value you use as `Bearer` token. The response remains also the same.
+
+```har
+{
+  "method": "GET",
+  "url": "https://${account.namespace}/api/v2/users/USER_ID",
+  "headers": [{
+    "name": "Authorization",
+    "value": "Bearer YOUR_MGMT_API_ACCESS_TOKEN"
+  }]
+}
+```
+
+### Use cross-origin authentication
+
+(to be updated)
+
+## Changes in Account Linking
+
+We are introducing some changes in the Account Linking process, one of which is breaking.
+
+
+You can no longer invoke the endpoint with the authenticated primary account's ID Token in the `Authorization` header. You have to use an Access Token which must include the `update:current_user_identities` scope. We have already seen how you can get an Access Token. In the following snippets you can see what changes in the invocation of the endpoint (that is, only the value you use in the `Authorization` header).
```

Comment:
Removing `id_tokens` as auth is what this migration is about. No need to highlight the case for Account Linking. 
The item here should be that we will only support using `user_id` in payload for AT with scope `update:users`. No other scopes will be allowed when that payload is present.